bertopic/plotting/_hierarchy.py

```diff
@@ -9,6 +9,7 @@
 import plotly.graph_objects as go
 import plotly.figure_factory as ff
 
+from .._utils import validate_distance_matrix
```

Comment:
```suggestion
from bertopic._utils import validate_distance_matrix
```
Please import it as above.